README.md

```diff
@@ -46,6 +46,8 @@ Flags:
       --accounts                            Export per account metrics
       --gatewayz                            Export gateway metrics
       --jsz string                          Export jsz metrics optionally, one of: all|streams|consumers
+      --jsz-leaders-only bool               Fetch jsz data from stream and consumer leaders only
+      --jsz-filter string                   Export jsz consumer metrics that match selected filters
```

Comment:
How users supposed to know what filters are?
If you edited this by hand: cobra probably generates possible values here?

Reply:
good point, this can be added to the readme since there would be a few...

Reply:
This is what we want:
https://github.com/TheDiveO/enumflag

Since there is no out of the box support: https://github.com/spf13/pflag/issues/236

Reply:
I've used it in the past, and it works well. A little bit initial setup, but it works way better than allowing strings here.

Reply:
sounds good to me (in case you want to give it a try in a follow up PR)